src/simplewallet/simplewallet.cpp

```diff
@@ -3217,18 +3218,29 @@ simple_wallet::simple_wallet()
                            boost::bind(&simple_wallet::print_locked_stakes, this, _1),
                            tr(USAGE_PRINT_LOCKED_STAKES),
                            tr("Print stakes currently locked on the Service Node network"));
+
+  char const OWNER_KEY_EXPLANATION[] = "By default this is the public key of an ed25519 keypair derived using the wallet's secret spend key as the seed value. ";
+  char const AVAILABLE_LNS_RECORDS[] = "You care currenly only able to purchase Session mappings. ";
   std::stringstream stream;
-  stream << "Buy a Loki Name Service mapping. Specifying `owner` is optional and defaults to the purchasing wallet if "
-            "empty or not specified. The `owner` should be a ed25519 public key; by default this is the public key of "
-            "an ed25519 keypair derived using the wallet's secret spend key as the seed value. You are currently only "
-            "able to purchase Session mappings\n\n";
+  stream << "Buy a Loki Name Service mapping. Specifying `owner` is optional and defaults to the purchasing wallet if empty or not specified. The `owner` should be a ed25519 public key. "
+            << OWNER_KEY_EXPLANATION
+            << AVAILABLE_LNS_RECORDS << "\n\n";
   stream << "Session: (max: " << lns::SESSION_DISPLAY_NAME_MAX << " bytes) map a human readable name to a Session public key.\n";
 
   m_cmd_binder.set_handler("buy_lns_mapping",
                            boost::bind(&simple_wallet::buy_lns_mapping, this, _1),
                            tr(USAGE_BUY_LNS_MAPPING),
                            tr(stream.str().c_str()));
 
+  stream.clear();
+  stream << "Update a Loki Name Service mapping's value field in the name->value mapping, you must be the owner of the the mapping by providing a signature that can be verified by the owner's public key."
```

Comment:
This message should mention that the signature is only needed if the registration didn't use the default.

But also: if I created using some other key, what value do I need to sign here?

Reply:
I've updated with the extra text
> The signature is derived using libsodium generichash on the {current txid blob, new value blob} of the mapping to update. Signature is an optional field and is signed using the wallet's spend key as an ed25519 keypair if it is not specified.

Reply:
This is going to get really technical when we want people to do this with their wallet key.  We're probably going to need some sort of "give me the hash" option here, which you can then plug into some menu in Session to generate the signature.  (But this can come later).